docker/setup/conda-helper.py

```
@@ -0,0 +1,462 @@
+    elif prefix.is_dir() and not unlocked:
+        missing_lockfiles = (
+            str(path.relative_to(directory)) for path in expected_lockfiles
+        )
+        missing_lockfiles = ", ".join(missing_lockfiles)
+        raise click.ClickException(f"Missing lockfiles: {missing_lockfiles}")
+    else:
+        locked = False
+
+    # Read environment.yaml
+    with click.open_file(file) as fd_env:
+        data_env = yaml.load(fd_env, yaml.Loader)
```

Ruff (0.16.1)

[error] Probable use of unsafe loader `Loader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)